src/DateTime/tests/test_datetime.py

```diff
@@ -683,6 +683,21 @@ def test_security(self):
         self.assertEqual(dt.__roles__, None)
         self.assertEqual(dt.__allow_access_to_unprotected_subobjects__, 1)
 
+    @unittest.skipUnless(PY3K, 'format method is Python 3 only')
+    def test_format(self):
+        dt = DateTime()
+        fmt = '%-d.%-m.%Y %H:%M'
+        result = dt.strftime(fmt)
+        self.assertEqual(result, '{:%-d.%-m.%Y %H:%M}'.format(dt))
+        self.assertEqual(str(dt), '{:}'.format(dt))
+        self.assertEqual(str(dt), '{}'.format(dt))
+        if sys.version_info > (3, 6, 0):  # pragma: no cover
```

Comment:
By default we run the coverage tests on Python 3, so the `no cover` is not needed here. I'd suggest to use the `PY3` mark, so this can be implemented without `eval` when Python 2 support gets dropped.
```suggestion
        if sys.version_info > (3, 6, 0):  # PY3
```